backend/api/Areas/Project/Models/Snapshot/ProjectReportModel.cs

```diff
@@ -0,0 +1,42 @@
+using System;
+
+namespace Pims.Api.Areas.Project.Models.Report
+{
+    /// <summary>
+    /// ProjectReportModel class, provides a model to represent the project reports.
+    /// </summary>
+    public class ProjectReportModel : Api.Models.BaseModel
+    {
+        #region Properties
+        /// <summary>
+        /// get/set - The primary key provides a unique identity for the project report.
+        /// </summary>
+        public int Id { get; set; }
+
+        /// <summary>
+        /// get/set - Whether this report is final and should not be deleted.
+        /// </summary>
+        public bool IsFinal { get; set; }
+
+        /// <summary>
+        /// get/set - An optional, descriptive name given to the report.
+        /// </summary>
+        public string Name { get; set; }
+
+        /// <summary>
+        /// get/set - The oldest date of comparison snapshots used in the report.
+        /// </summary>
+        public DateTime? From { get; set; }
+
+        /// <summary>
+        /// get/set - The most recent date of comparison snapshots used in the report.
+        /// </summary>
+        public DateTime? To { get; set; }
```

Comment:
I ended up doing this because I wanted the backend to generate the To value initially, but can go the other way if you like.

Reply:
Ah yes I see, good point.  It can be `null` for the model then.  One of those scenarios where the model would be different depending on whether it's a response or request.

Reply:
The other issue with non-nullable dates is that if you don't pass in a date, .net core substitutes in a default time:
1/1/0001 12:00:00 AM
Explicitly checking for this is a lot messier than checking for null in my opinion.